yt/frontends/arepo/data_structures.py

```diff
@@ -56,9 +56,8 @@ def _is_valid(self, *args, **kwargs):
                 )
             )
             fh.close()
-        except:
+        except Exception:
```

Comment:
Oof, I'm glad we're adding `Exception` but we should probably improve this.  (I know you made this easier to improve in your load refactor work.)

Reply:
Yes, it's also one of the goals on this PR: https://github.com/yt-project/yt/pull/2760 !